server.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./dist/server.js")
```

Comment:
I'll add a comment. The summary is that I want to use this module as part of my test suite instead of running it locally on my laptop when doing manual testing of the website.

This is useful for CI where I want to control the port.

Reply:
Running it locally makes sense.  That was the idea behind the `npm run run` script.  The port can be set with an env var so we could do something so that it inherits the port if it's already set: `"run": "LOG_LEVEL=${LOG_LEVEL:=true} PORT=${PORT:=true} node dist/index.js"` and then you could run it like `PORT=8001 npm run run`.